gitpod-io/gitpod
# Simplify example repositories

## Description

Make the logic less error-prone and fix an issue with some demo repos being displayed twice.

## Related Issue(s)

Fixes ENT-725

## How to test

https://ft-simplif19178b7d05.preview.gitpod-dev.com/workspaces

1. Open a workspace with the docker example which is by default listed in the repo suggestions.
2. Open the list again and start searching.
3. Make sure of two things: 1) if the search string matches the docker example repo, it's only listed once and 2) if it doesn't match, it's not displayed at all
